src/TLuaInterpreter.cpp

```diff
@@ -373,11 +372,8 @@ int TLuaInterpreter::raiseEvent( lua_State * L )
             event.mArgumentTypeList.append( ARGUMENT_TYPE_NIL );
         }
         else {
-            lua_pushstring( L, tr( "raiseEvent: bad argument #%1 type (string, number, boolean, or nil\n"
-                                   "expected, got a %2!)" )
-                            .arg( i )
-                            .arg( luaL_typename( L, i ) )
-                            .toUtf8().constData() );
+            lua_pushfstring(L, "raiseEvent: bad argument #%d type (string, number, boolean, or nil\n"
```

Comment:
I guess I'm not sure why we are hardwrapping an error message like this, but I suppose it's not changed, so no need to fix it.